x-pack/plugins/security_solution/public/common/components/links/index.tsx

```diff
@@ -114,7 +120,6 @@ export const ExternalLink = React.memo<{
       <EuiToolTip content={url} position="top" data-test-subj="externalLinkTooltip">
         <EuiLink href={url} target="_blank" rel="noopener" data-test-subj="externalLink">
           {children}
-          <ExternalLinkIcon data-test-subj="externalLinkIcon" />
           {!isNil(idx) && idx < lastIndexToShow && <Comma data-test-subj="externalLinkComma" />}
```

Comment:
I think we can also move 
```
{!isNil(idx) && idx < lastIndexToShow && <Comma data-test-subj="externalLinkComma" />}
```
out of `EuiLink` , so the comma can stay on the right hand side of the icon.
![Screenshot 2021-03-11 at 02 32 49](https://user-images.githubusercontent.com/6295984/110679146-0e1e1d00-81cf-11eb-8adb-cac7ad85c5c8.png)